src/phantom.js

```diff
@@ -71,6 +72,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
                 /* Protocol-less URL */
                 url = 'http://' + originalUrl.substr(7);
             }
+            console.log('After: ' + url);
```

Comment:
Debugging information? There's a couple console.log()'s in there... Should either remove them, or use `require('debug')`, or another logging method.